### PR TITLE
Random documentation improvements

### LINKED ISSUE #1426: compare_reqs gives incorrect path for rebuild_pipenv script
See: https://github.com/nucypher/nucypher/blob/master/scripts/circle/compare_reqs.sh#L19

This script is no longer `installation` and appears to have been moved to `circle`.

### LINKED ISSUE #2048: Standalone language audit of `literature.py`
After establishing a guideline for inclusion in literature, let's have an individual(s) comb through this module for accuracy, consistent style, voice, and tone.  Consider the possibility of a language translation (we have a multi-national team collectively speaking at least 3 languages).

From: <https://www.umgc.edu/current-students/learning-resources/writing-center/online-guide-to-writing/tutorial/chapter3/ch3-21.html>

>- Style means the mechanical or technical aspects of writing and may be specific to the requirements of the subject or topic.
>- Voice means the unique worldview and word choices of the author.
>- Tone means the attitude conveyed in the writing and may encompass formality, objectivity, intimacy, and similar aspects.

### LINKED ISSUE #2054: Update `Contributing` guide for building of documentation on MacOS
`make docs` is for Linux, but `make mac-docs` is for MacOS. We should explain that. Related to #1931 .

It may be best to adjust `make docs` based on platform (if possible)...?

### LINKED ISSUE #2136: Installation Docs are Innacurate (pip install -e.[development])
There is no longer an extra `development` it is simply `dev`. (see <https://docs.nucypher.com/en/latest/guides/installation_guide.html#pip-development-installation>)

### PR DESCRIPTION
**Pull request by derekpierre**
- [x] Fixes #1426 
- [x] Fixes #2048  (Audit of `literature.py` for common style & tone)
- [x] Fixes #2054 
- [x] Fixes #2136

### CODE REVIEW

**Review by mswilkison — APPROVED** [review]

**Review by vzotova — APPROVED** [review]

**Review by arjunhassard — APPROVED** [review]
Looks good! ✅

File: Makefile [hunk 1]
```diff
@@ -31,14 +31,14 @@ build-docs:
 	$(MAKE) -C docs html
 
 validate-docs: build-docs
-    # Required dependencies from docs/requirements.txt
+    # Required dependencies from docs-requirements.txt
```
Comment (KPrasch) [inline Makefile:34]:
```suggestion 
    # Requires dependencies from docs-requirements.txt
```

File: nucypher/cli/literature.py [hunk 2]
```diff
@@ -106,7 +106,7 @@
 
 SELECT_STAKING_ACCOUNT_INDEX = "Select index of staking account"
 
-NO_ACTIVE_STAKES = "There are no active stakes\n"
+NO_ACTIVE_STAKES = "No active stakes found\n"
```
Comment (KPrasch) [inline nucypher/cli/literature.py:109]: much better :+1:

File: nucypher/cli/literature.py [hunk 3]
```diff
@@ -423,7 +423,7 @@
 
 SUCCESSFUL_RETARGET = "Successfully re-targeted {contract_name} proxy to {target_address}"
 
-SUCCESSFUL_RETARGET_TX_BUILT = "Transaction to retarget {contract_name} proxy to {target_address} was built:"
+SUCCESSFUL_RETARGET_TX_BUILT = "Successfully built transaction to retarget {contract_name} proxy to {target_address}:"
```
Comment (KPrasch) [inline nucypher/cli/literature.py:426]: It's almost worth documenting the thought process and tone used throughout here.

File: Makefile [hunk 4]
```diff
@@ -31,14 +31,14 @@ build-docs:
...
 	python newsfragments/validate_files.py
 	towncrier --draft
 
 docs: build-docs validate-docs
 	readlink -f docs/build/html/index.html
 
-mac-docs: build-docs
+mac-docs: build-docs validate-docs
```
Comment (KPrasch) [inline Makefile:41]: It looks like validate includes build-docs: `validate-docs: build-docs` -- so this seems redundant

Comment (derekpierre) [reply Makefile:41]: @KPrasch and I chatted about it - leaving it as is

File: nucypher/cli/literature.py (hunk 3 above)
Comment (derekpierre) [reply nucypher/cli/literature.py:426]: Haha really it was just from the top of the file to the bottom i.e. the style used higher up in the file, was replicated lower down in the file.

**Pull request merged**